run.py

```diff
@@ -2,10 +2,13 @@
 import logging
 import os
 
+from bot.store import createStorageObject
+
 def main():
     logging.basicConfig(level=logging.INFO)
     logging.info("Running")
 
+    createStorageObject() # exception if no storage
```

Comment:
Naming: perhaps you would consider `ensureStorageObject()` as the idea is that this function acts as a fence to prevent the code from continuing further if misconfigured.